server/storage/schema/schema.go

```diff
@@ -129,10 +137,21 @@ var (
 	// schema was introduced in v3.6 as so its changes were not tracked before.
 	schemaChanges = map[semver.Version][]schemaChange{
 		version.V3_6: {
-			addNewField(Meta, MetaStorageVersionName, emptyStorageVersion),
+			// emptyValue is used for v3.6 Step for the first time, in all other version StoragetVersion should be set by migrator.
+			addNewField(Meta, MetaStorageVersionName, emptyValue),
+		},
+		version.V3_5: {
+			// UnsafeReadConsistentIndex checks for zero-length values
+			addNewField(Meta, MetaTermKeyName, emptyValue),
+			// UnsafeConfStateFromBackend will fail on []byte(""), use empty struct as default
+			// note: UnsafeConfStateFromBackend is never used in 3.5, in 3.6 it's only used in `migrate`
```

Comment:
Don't understand this note. Can you elaborate?

Reply:
I was wondering if setting empty struct as default for `MetaConfStateName` will cause any issues when running 3.5 after upgrade from 3.4. So, I've checked for [usage](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/etcd-io/etcd%24+rev:release-3.5+UnsafeConfStateFromBackend&patternType=standard&sm=0&groupBy=path) of `UnsafeConfStateFromBackend` in release-3.5 branch and didn't find any.